docs/environments/list-environments.md

```diff
@@ -0,0 +1,20 @@
+---
+title: List Environments
+intro: Access the list of project environment to see details of an environment.
```

Comment:
```suggestion
intro: You can find and view details of project environments you have access to
```

Reply:
intro updated to better describe the post objective